pkg/loop/plugin_test.go

```diff
@@ -124,14 +124,20 @@ func TestHelperProcess(t *testing.T) {
 		}
 	}
 
+	lggr, err := loop.NewLogger()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to create logger: %s\n", err)
+		os.Exit(2)
```

Comment:
Why exit code 2?

Reply:
I had the same thought, but I'm not sure. Just following local style.